### ISSUE TITLE
Importing local modules seems to break the code lenses

### ISSUE THREAD

**Issue by phated** [grain-lang/grain-language-server#22]
I was trying to abstract my Advent of Code stuff, so I split their input into a new file and it ends up breaking my type hinting.

This is my file:
```
import Array from 'array';

import { data } from './data';
// import { data } from './sample-data

let allPairs = Array.product(data, data);

let (a, b) = Array.find(((a, b)) => a + b == 2020, allPairs);
print((a, b));

print(a * b);
```

Where `data` is just an array of numbers

**Comment by ng-marcus** [comment]
Thanks for reporting, I'll take a look today.

**Comment by ng-marcus** [comment]
I took your code and get type hints.  I did need to press return to trigger a pass of the compiler though, there's no updating based on dependencies yet.    The lenses disappear when there's a compilation error as currently the grain compiler exits with an exception when there's a compilation error, and I can't be sure if any of the lenses are still valid at that point


![image](https://user-images.githubusercontent.com/20163757/101258086-f9d20c00-3717-11eb-9dc3-ed8fe6afe3ac.png)

Let me know what you're seeing though, and I'll try and reproduce.

**Comment by phated** [comment]
@ng-marcus If you have a chance, can you check against https://github.com/phated/grain-advent-of-code - I'm having lots of trouble with the code lenses against that project.

**Comment by ng-marcus** [comment]
Thanks, I've reproduced the problem now.    It's how the compiler is locating relative files when I pass an absolute path.

**Comment by ng-marcus** [comment]
Fixed with #30 

Let me know if that fixes it for you so I can close this issue.
